src/guiguts/tools/jeebies.py

```diff
@@ -16,6 +16,16 @@
 _the_jeebies_checker = None  # pylint: disable=invalid-name
 
 
+########################################################
+# jeebies.py
+# C author: Jim Tinsley (DP:jim) - 2005
+# Go author: Roger Franks (DP:rfrank) - 2020
+# Python author: Quentin Campbell (DP:qgc) - 2024
+########################################################
+# 2024.04.06 Used sentence case with full stop for first
+#            sentence of docstrings. Ditto for args.
```

Comment:
Although I'm not against having a record of the original authors of these tools, I don't think we want to add comments about all the other changes we make (like the docstring capitalization). Rather than looking in the code comments to see what changes have been made recently, Github provides us with several mechanisms to help do that, such as the PR comments/conversations, the commit comments, and "blame" view (e.g. see [this blame view](https://github.com/DistributedProofreaders/guiguts-py/blame/master/src/guiguts/maintext.py) of maintext.py where you can see who edited each line of code and why)
So, I'd prefer to see these two comment lines removed, please.

Reply:
The docstrings look great now, thank!